src/views/resultView.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 import {
-  USER_INTERFACE_ID,
+  // USER_INTERFACE_ID,
   CLOSE_BUTTON_ID
  } from '../constants.js';
  import {ShowRightAnswer} from '../views/answerView.js'
 
-
-const userInterface = document.getElementById(USER_INTERFACE_ID)
-userInterface.innerHTML = '';
-
-export const createResultMessage = () => {
+ 
+ export const createResultPage = () => {
   const resultMessage = document.createElement('div');
   resultMessage.innerHTML = `
-  <h1> Your score is ${ShowRightAnswer} from 10 </h1>
-  <button id="${CLOSE_BUTTON_ID}">Close</button>
+  <h1> Your score is 
```

Comment:
You still need to put in the final score here.

Reply:
Actually I tried to add the final score here but it doesn't work so that's why I moved it out and comet it.
It's in the line 18